### PR TITLE
printer

### PR DESCRIPTION
**Pull request by XWasHere**
B)

### CODE REVIEW

**Comment by EmeraldSnorlax** [comment]
when the contribution role is being farmed

File: printer/index.html [hunk 1]
```diff
@@ -19,20 +19,26 @@
     Your browser does not support the audio element.
     </audio>
 
-    <button onclick="playAudio()" type="button">printer</button>
-    <button onclick="pauseAudio()" type="button">printer stop</button> 
+    <button onclick="doAudio()" type="button" id="audiobtn">printer</button>
```
Comment (EmeraldSnorlax) [inline printer/index.html:22]: changing the id away from `button` is probably a good idea

File: printer/index.html [hunk 2]
```diff
@@ -19,20 +19,26 @@
...
 
      <!--Javascript 😩-->
     <script>
     var x = document.getElementById("myAudio"); 
+    var y = document.getElementById("audiobtn");
```
Comment (EmeraldSnorlax) [inline printer/index.html:27]: when the variable names

**Comment by XWasHere** [comment]
i'm already contributor woe

**Comment by EmeraldSnorlax** [comment]
honestly -- what even is the point of using javascript here? browsers already support native audio playback with a controller -- simply add the `controls` keyword to the tag, like so:

```html
    <audio controls id="myAudio">
    <source src="../media/printer.mp3" type="audio/mpeg">
    Your browser does not support the audio element.
    </audio>
```

im going to leave this pr open should @SoNothingMC want to comment, but using javascript here seems silly.

**Comment by XWasHere** [comment]
this is *big*rat, it can't be simple.

**Comment by EmeraldSnorlax** [comment]
why use js when its not needed 
its:

- easier to maintain without js
- more user friendly without js
- more usable for people with assistive technologies without js
- less data needs to be transferred to the client without js
- isnt required for it to function (browsers have already implemented native audio playback with controls)

**Comment by XWasHere** [comment]
maybe, but i don't know if people want to see a bunch of controls.

this is literally a page that makes printer noises

**Comment by bigratenthusiast** [comment]
JS is swaggy asf 💪 🔥 🔥 💯 ✈️ #MondayMotivation #SigmaGrindset

**Comment by d1snin** [comment]
no, pls leave this pr conversation

**Review by d1snin — APPROVED** [review]

**Comment by XWasHere** [comment]
"dead chat eks dee"

**Comment by EmeraldSnorlax** [comment]
> "dead chat eks dee"

I'll merge for now, but I still think that @SoNothingMC should weigh in about the use of JavaScript at all.

**Comment by d1snin** [comment]
:trollface:

**Comment by smokeytube** [comment]
when the pull request sus

**Comment by d1snin** [comment]
when the pull request sus

**Comment by EmeraldSnorlax** [comment]
no no no no nope not again nope no no

**Pull request merged**